### DISCUSSION TITLE
What's the difference of the tags in TagStudio?

### DISCUSSION THREAD

**Discussion by CoreOptd** [TagStudioDev/TagStudio#340] [Q&A]
![image](https://github.com/user-attachments/assets/ae7dfe84-6671-4f00-ba05-e3de6631ab86)
When adding a field to an image, I noticed 3 different tag types: **Tags, Content Tags and Meta Tags.** And I was wondering what the differences are;

- **do they serve different purposes** and do it in different ways
- or **are they just for the sake of organisation** (meaning it's just 3 different tag boxes that are renamed and are not at all different from one another)?

If it's the latter, why was the need to add 3 different tag boxes, is there a limit to how much tags you could add or something (it's the only real reasoning I could really find)

**Comment by samuellieberman** [comment]
@CoreOptd They are purely for the sake of organization on an entry-by-entry basis. In the future, CyanVoxel has expressed that he plans to make a tag category property associated with the tags themselves. That means that in the future, it won't be possible to add the same tag to different categories, and it won't be necessary to manually different tag boxes to entries. Personally, I just shove everything into a "Tags" box and forget about the others.
